SonarLint.VisualStudio.Integration.sln

```diff
@@ -162,6 +162,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SonarQube.Client", "src\Son
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SonarQube.Client.Tests", "src\SonarQube.Client.Tests\SonarQube.Client.Tests.csproj", "{4F5AE2A8-56DD-44EC-9872-B7884CC1C1DB}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "VsAnalyzerConfig", "VsAnalyzerConfig", "{682C440B-E506-4811-9569-9734D8DF4620}"
+	ProjectSection(SolutionItems) = preProject
+		src\vs-threading.MainThreadAssertingMethods.txt = src\vs-threading.MainThreadAssertingMethods.txt
+		src\vs-threading.MainThreadSwitchingMethods.txt = src\vs-threading.MainThreadSwitchingMethods.txt
+	EndProjectSection
+EndProject
```

Comment:
Added the VSSDK analyzer config files to the solution to make them easier to find.